analytics.go

```diff
@@ -57,7 +57,7 @@ type client struct {
 
 	// This HTTP client is used to send requests to the backend, it uses the
 	// HTTP transport provided in the configuration.
-	http http.Client
+	http *http.Client
 }
```

Comment:
Interesting, I think I originally just copied what I saw here https://github.com/google/go-github/blob/master/github/github.go#L74 and have been using that since!